app/views/receipt_mailer/receipt_email.html.erb

```diff
@@ -37,6 +37,7 @@ The information we have on record is listed below.</p>
 <strong>Abstract:</strong><br>
 <%= simple_format(@thesis.abstract, {}, wrapper_tag: "p") %>
 <strong>Notes:</strong> <%= @thesis.author_note %><br>
+<strong>Consent to send thesis to ProQuest:</strong> <%= render_proquest_status(@thesis) %>
```

Comment:
Will we need logic to prevent this from showing up in the email for bachelor's degrees?

Reply:
Good point. This reminds me that, ideally, we also want logic to prevent the field from appearing in `thesis/new` for bachelor's degrees. If I were an undergrad who opted in to PQDT, I'd be confused when my thesis never made it in there. That said, I'm not sure how that would work without prefilled degree data, and hopefully it's a very small number of submissions that would be affected.

Reply:
Yeah we had that `new` form out of scope for this work (at least for now).